README.md

```diff
@@ -137,8 +135,6 @@ static class AndroidJavaCompile_BootClasspath_Workaround implements Workaround {
 
 Fixed by the Android Cache Fix plugin, but unresolved in any current or upcoming preview release of the Android Gradle Plugin:
 
-* Room annotation processor causes cache misses, doesn't declare outputs, overlapping outputs, etc: https://issuetracker.google.com/issues/132245929
```

Comment:
Can we keep a reference to this issue in the readme and also explain that if they encounter this issue, they should use the Room Gradle plugin?

Reply:
added some context [3eff77c](https://github.com/gradle/android-cache-fix-gradle-plugin/pull/601/commits/3eff77c2fdf708736f006a1690697f2c52b858ba) 
wdyt?